### PR TITLE
Remove work_packages_duration_field_active feature flag

### PR DESCRIPTION
**Pull request by oliverguenther**

### CODE REVIEW

File: app/services/work_packages/set_schedule_service.rb [hunk 1]
```diff
@@ -174,16 +174,14 @@ def schedule_on_missing_dates(scheduled, min_start_date)
               scheduled.due_date && scheduled.due_date < min_start_date ? min_start_date : scheduled.due_date)
   end
 
-  def reschedule_by_delta(scheduled, moved_delta, min_start_date, dependency)
-    days = WorkPackages::Shared::Days.for(dependency.work_package)
+  def reschedule_by_delta(scheduled, moved_delta, min_start_date)
+    days = WorkPackages::Shared::Days.for(scheduled)
```
Comment (dombesz) [inline app/services/work_packages/set_schedule_service.rb]: At first sight this change seems to be unrelated to removing the flag. Is this intended to be part of the PR?

Comment (cbliard) [reply app/services/work_packages/set_schedule_service.rb]: True, this is not related. That's something I could have done in another PR. Should I remove it?

Comment (dombesz) [reply app/services/work_packages/set_schedule_service.rb]: If it fails no tests, I'm fine with it. Perhaps do it in a separate commit?

Comment (cbliard) [reply app/services/work_packages/set_schedule_service.rb]: I just removed it from this PR. I'll add it in my other PR. Thanks for the feedback.

**Review by dombesz — APPROVED** [review]
